lib/stdlib/src/maps.erl

```diff
@@ -89,6 +143,44 @@ keys(_) -> erlang:nif_error(undef).
 
 merge(_,_) -> erlang:nif_error(undef).
 
+-spec merge(Combiner, Map1, Map2) -> Map3 when
+    Map1 :: #{Key1 => Value1},
+    Map2 :: #{Key2 => Value2},
+    Combiner :: fun((Key1, Value1, Value2) -> CombineResult),
```

Comment:
```suggestion
    Combiner :: fun((Key, Value1, Value2) -> CombineResult),
```

Reply:
Can you explain why you think this change is an improvement? I think the original type is more restricted.

Reply:
Sorry, I misread the spec.